public/js/dashboard.js

```diff
@@ -0,0 +1,13 @@
+const blogTableSelections = []
```

Comment:
change this to
```js
var blogTableSelections = [] // eslint-disable-line
```
This should disable the linter.

Reply:
👍